eos-phone-home

```diff
@@ -70,13 +70,17 @@ class PhoneHome(object):
 
     ACTIVATION_ENDPOINT = os.getenv('EOS_PHONE_HOME_ACTIVATE_URL',
                                     "https://home.endlessm.com/v1/activate")
-    ACTIVATION_VARIABLES = ['dualboot', 'live', 'image', 'release',
-                            'vendor', 'product', 'serial']
+    ACTIVATION_VARIABLES = {'dualboot', 'live', 'image', 'release',
+                            'vendor', 'product', 'serial'}
```

Comment:
If you're changing the type of this collection, I would rather this be `('dualboot', ...)` (a tuple) or `frozenset(('dualboot', ...))` since it is meant to be immutable.

Reply:
👍